system-tests/services/test-pods.js

```diff
@@ -1094,5 +1094,196 @@ describe("Services", function() {
+          env: {
+            camelCase: "test",
+            snake_case: "test",
+            lowercase: "test",
+            UPPERCASE: "test"
+          }
+        }
+      ]);
+
+      cy.get("button").contains("Review & Run").click();
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Service ID")
+        .contains(`/${Cypress.env("TEST_UUID")}/${serviceName}`);
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Instances")
+        .contains("1");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("CPU")
+        .contains("0.1 (0.1 container-1)");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Memory")
+        .contains("10 MiB (10 MiB container-1)");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("Disk")
+        .contains("Not Supported");
+
+      cy
+        .root()
+        .configurationSection("General")
+        .configurationMapValue("GPU")
+        .contains("Not Supported");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Container Image")
+        .contains("Not Configured");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Force Pull On Launch")
+        .contains("Not Configured");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("CPUs")
+        .contains("0.1");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Memory")
+        .contains("10 MiB");
+
+      cy
+        .root()
+        .configurationSection("Containers")
+        .configurationMapValue("Command")
+        .contains(command);
+
+      cy
+        .root()
+        .configurationSection("Environment Variables")
+        .then(function($envSection) {
+          expect($envSection.get().length).to.equal(1);
+
+          const $tableRows = $envSection.find("tbody tr:visible");
+          const cellValues = [
+            ["camelCase", "test", "Shared", "Edit"],
+            ["snake_case", "test", "Shared", "Edit"],
+            ["lowercase", "test", "Shared", "Edit"],
+            ["UPPERCASE", "test", "Shared", "Edit"]
+          ];
+
+          $tableRows.each(function(rowIndex) {
+            const $tableCells = cy.$(this).find("td");
+
+            expect($tableCells.length).to.equal(4);
+
+            $tableCells.each(function(cellIndex) {
+              expect(this.textContent.trim()).to.equal(
+                cellValues[rowIndex][cellIndex]
+              );
+            });
+          });
+        });
+
+      cy.get("button").contains("Run Service").click();
+
+      cy.get(".page-body-content table").contains(serviceName).should("exist");
+
+      cy
```

Comment:
@orlandohohmeier @juliangieseke - please make sure we extend this to include a configuration check too. Thanks